common/ast/tests/it/parser.rs

```diff
@@ -186,6 +186,7 @@ fn test_query() {
     let mut mint = Mint::new("tests/it/testdata");
     let mut file = mint.new_goldenfile("query.txt").unwrap();
     let cases = &[
+        r#"select * from a limit 3 offset 4 format csv"#,
```

Comment:
@sundy-li  where supposed to use the format in this SQL?

Reply:
At the end of the sql.